src/lib.rs

```diff
@@ -185,6 +194,7 @@ impl QrCode {
 
     /// Checks whether a module at coordinate (x, y) is a functional module or
     /// not.
+    #[cfg(feature = "std")]
```

Comment:
Note that this is the only function actually enabled with `std`. All other `std` usage is already behind `svg` and `image` flags.